lib/src/model/assertion.dart

```diff
@@ -65,7 +65,7 @@ class Assertion {
 
```

Comment:
@Rishabhmodii add a copoywith to assertion class, similar to copy function of go version.